### DISCUSSION TITLE
commands for the minecraft server

### DISCUSSION THREAD

**Discussion by thermospud** [TAServers/issues#31] [Addon Suggestions]
- /sethome saves your current position to be used with /home
- /home teleports to the saved position

i think these commands could be useful for players going on long travel expeditions to be able to go back home, how this could be useful is that players who will go on long travel expeditions will usually carry beds so they don't lose their progress, after they reach their destination if their base is a long way from spawn it will be even more tedious to get back. and /sethome will not have a name parameter (multiple homes) for simple balancing

**Comment by yogwoggf** [comment]
Exists already:

step 1: Create a guild if you haven't with `/guild create <name here>`
step 2: Walk to your **starting position that you want to be teleported to.**
step 3: Type into chat: `/guild settings hq set`

Now you should be able to type `/guild hq` to be teleported back to where you set the HQ

**Reply by thermospud** [reply]
Yes that is true, but then when people do this wouldn't it clutter up the Guilds list for people serious about Guilds? plus its way more noob-friendly by just having 1 setup command i would suppose. idk

**Reply by yogwoggf** [reply]
You don't have to be "serious" about guilds you can join/leave them as you please. The entire home idea is inconsistent with the claim and guild mod so really you'd be better off talking about Argonauts and Cadmus (our guild & claim mod respectively)

**Reply by thermospud** [reply]
fair enough
